source/Octopus.Client/Util/ParameterHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Octopus.Client.Util
+{
+    public static class ParameterHelper
+    {
+        public static Dictionary<string, object> CombineParameters(object leftHandSide, object rightHandSide)
```

Comment:
We might want to be a little bit more specific here about precedence. If the user supplies a `includeGlobal` parameter to a request made within a repository that is already scoped to a certain space context, the repositories context should "win" or overwrite whatever the user has supplied.